src/main.rs

```diff
@@ -121,16 +121,43 @@ impl Results {
         }
     }
 
-    fn update(&mut self, case: &TestCase, index: Option<u32>, result: &TestResult) {
+    //fn update(&mut self, case: &TestCase, index: Option<u32>, result: &TestResult) {
```

Comment:
Remove

Reply:
For formatting I recommend running [`rustfmt`](https://github.com/rust-lang-nursery/rustfmt)

Reply:
fixed and formatted.